examples/compose/plot_compare_reduction.py

```diff
@@ -102,13 +102,11 @@
 #     cache. Hence, use the ``memory`` constructor parameter when the fitting
 #     of a transformer is costly.
 
-from tempfile import mkdtemp
-from shutil import rmtree
 from joblib import Memory
 
 # Create a temporary folder to store the transformers of the pipeline
-cachedir = mkdtemp()
-memory = Memory(location=cachedir, verbose=10)
+location = './cachedir'
```

Comment:
Just `"cachedir"` for an OS independent path, otherwise LGTM. Thanks!

Reply:
Done